src/app.controller.ts

```diff
@@ -21,4 +23,13 @@ export class AppController {
         console.log(this.configService.getConfig().server);
         return this.configService.getConfig();
     }
+
+    @Get("/redis/:key/:val")
```

Comment:
Controller segmentation should be considerated

Reply:
changed